lib/diaspora_federation/discovery/web_finger.rb

```diff
@@ -148,7 +148,7 @@ def to_xml
       # @return [WebFinger] WebFinger instance
       # @raise [InvalidData] if the given XML string is invalid or incomplete
       def self.from_xml(webfinger_xml)
-        data = parse_xml_and_validate(webfinger_xml)
+        data = parse_and_validate(webfinger_xml)
```

Comment:
I think we should keep that (because it is really "parse xml" here and it is a special case for webfinger), but it's probably unintentionally because of search/replace?